tests/test_cli.py

```diff
@@ -294,3 +294,18 @@ def test_mlflow_artifact_service_unavailable_without_config():
         )
     finally:
         process.kill()
+
+
+def test_mlflow_artifact_only_raises_with_invalid_backend_store_uri():
+
+    with mock.patch("mlflow.server._run_server") as run_server_mock:
+        result = CliRunner().invoke(
+            server,
+            ["--serve-artifacts", "--artifacts-only", "--backend-store-uri", "sqlite:///my.db"],
+            catch_exceptions=False,
+        )
+        assert result.output.startswith(
```

Comment:
```suggestion
        assert result.stderr.startswith(
```

Can we use `stderr` here?